svc-bie-kafka/src/main/resources/application.yaml

```diff
@@ -14,21 +14,28 @@ spring:
   kafka:
     bootstrap-servers: "${BIE_KAFKA_PLACEHOLDERS_BROKERS:localhost:9092}"
     properties:
-      schema.registry.url: "${BIE_KAFKA_PLACEHOLDERS_BROKERS:https://schemaregistry.dev.bip.va.gov:443}"
+      schema.registry.url: "${BIE_KAFKA_PLACEHOLDERS_BROKERS:https://USERNAME:PASSWORD@schemaregistry.dev.bip.va.gov:443}"
       specific.avro.reader: false
+      security.protocol: SASL_SSL
+      security.inter.broker.protocol: SASL_SSL
+      sasl:
+        mechanism: PLAIN
+        username: USERNAME
+        password: PASSWORD
+        jaas:
+          config: "org.apache.kafka.common.security.plain.PlainLoginModule required username=\"USERNAME\" password=\"PASSWORD\";"
     consumer:
       group-id: "${BIE_KAFKA_PLACEHOLDERS_GROUP_ID:vro-bie-tst-vro}"
       key-deserializer: "org.apache.kafka.common.serialization.StringDeserializer"
       value-deserializer: "org.apache.kafka.common.serialization.StringDeserializer"
       auto-offset-reset: earliest
       # Comment out the following line to connect without SSL
-      security.protocol: SSL
       ssl:
```

Comment:
is this needed if we are no longer using ssl for this?

Reply:
We are using SASL_SSL which still needs certs because it uses TLS encryption. The authentication method is defined as PLAIN, which will use the SSL encryption protocol to transmit a username:password. So we still will need the certs if we want to use SASL_SSL. There is an alternative to use SASL_PLAINTEXT which doesn't use TLS encryption, but it seems like the BIE folks want us to use SASL_SSL.

Reply:
Ah that makes sense. Thanks for clarifying. They have hinted that this is a replacement and not an addition to auth. But would be great to clarify with them.